core/primitives/src/types.rs

```diff
@@ -651,6 +651,35 @@ pub mod validator_stake {
                 stake_next_epoch: if is_next_epoch { self.stake() } else { 0 },
             }
         }
+
+        // TODO(chunk-validator-assignment) if `ValidatorStake` is recalculated every epoch, this should be a field of (new) `ValidatorStakeV2`?
```

Comment:
Let's make a github issue for this and use the issue number.

Reply:
Created #10014 and used it for the annotation in 452d745.